arduino-ide-extension/src/electron-main/theia/electron-main-application.ts

```diff
@@ -381,7 +375,7 @@ export class ElectronMainApplication extends TheiaElectronMainApplication {
     super.onWillQuit(event);
   }
 
-  get windows(): BrowserWindow[] {
-    return this._windows.slice();
+  get browserWindows(): BrowserWindow[] {
+      return this._windows;
```

Comment:
Hi @msujew, the super class already does the window tracking (See: https://github.com/eclipse-theia/theia/pull/10600). Why not drop the window tracking here, and rely on the Theia impl? 

I did this Theia version bump on another mac, if you think it is useful, I can push the changes in a couple of hours.

Reply:
Sure, If you want to, you can push your changes directly on top of mine :)

Reply:
Thank you, I'll create a PR against your branch later today and you can decide if you want it or not.